docs/src/dynamics.md

```diff
@@ -1,38 +1,39 @@
 # Dynamics
 
-This page summarizes the governing equations behind Breeze’s atmospheric dynamics and the anelastic formulation used by `AtmosphereModel`, following the thermodynamically consistent framework of Pauluis (2008) [Pauluis2008](@citet).
+This section summarizes the governing equations behind Breeze’s atmospheric dynamics and the anelastic formulation used by [`AtmosphereModel`](@ref), following the thermodynamically consistent framework of [Pauluis2008](@citet).
 
-We begin with the compressible Navier–Stokes momentum equations and reduce them to an anelastic, conservative form. We then introduce the moist static energy equation and outline the time-discretized pressure correction used to enforce the anelastic constraint.
+We begin with the compressible Navier–-Stokes momentum equations and reduce them to an anelastic, conservative form.
+We then introduce the moist static energy equation and outline the time-discretized pressure correction used to enforce the anelastic constraint.
 
 ## Compressible momentum equations
 
 Let ``ρ`` denote density, ``\boldsymbol{u}`` velocity, ``p`` pressure, ``\boldsymbol{f}`` non-pressure body forces (e.g., Coriolis), and ``\boldsymbol{\tau}`` subgrid/viscous stresses. With gravity ``g \hat{\boldsymbol{z}}``, the inviscid compressible equations in flux form are
 
 ```math
 \begin{aligned}
-&\text{Mass:} && \partial_t \rho + \nabla\!\cdot(\rho\,\boldsymbol{u}) = 0. \\
-&\text{Momentum:} && \partial_t(\rho\,\boldsymbol{u}) + \nabla\!\cdot(\rho\,\boldsymbol{u}\,\boldsymbol{u}) + \nabla p 
-\;=\; \rho\,g\,\hat{\boldsymbol{z}} + \rho\,\boldsymbol{f} + \nabla\!\cdot\boldsymbol{\tau}. 
+&\text{Mass:} && \partial_t ρ + \boldsymbol{\nabla \cdot}\, (ρ\,\boldsymbol{u}) = 0. \\
+&\text{Momentum:} && \partial_t(ρ\,\boldsymbol{u}) + \boldsymbol{\nabla \cdot}\, (ρ\,\boldsymbol{u}\,\boldsymbol{u}) + \boldsymbol{\nabla} p = ρ\,g\,\hat{\boldsymbol{z}} + ρ\,\boldsymbol{f} + \boldsymbol{\nabla \cdot}\, \boldsymbol{\tau}.
 \end{aligned}
 ```
 
 For moist flows we also track total water (vapor + condensates) via
 
 ```math
-\partial_t(\rho q^t) + \nabla\!\cdot(\rho q^t\,\boldsymbol{u}) = S_q ,
+\partial_t(ρ q^t) + \boldsymbol{\nabla \cdot}\, (ρ q^t\,\boldsymbol{u}) = S_q ,
 ```
 
 where ``q^t`` is total specific humidity and ``S_q`` accounts for sources/sinks from microphysics and boundary fluxes.
 
-Thermodynamic relations (mixture gas constant ``R^{m}``, heat capacity ``c^{pm}``, Exner function, etc.) are summarized on the Thermodynamics page.
+Thermodynamic relations (mixture gas constant ``R^{m}``, heat capacity ``c^{pm}``, Exner function, etc.)
+are summarized on the [Thermodynamics](@ref Thermodynamics-section) section.
```

Comment:
I'm not a native English speaker, but I think I say "in a section" rather than "on a section"?
```suggestion
are summarized in the [Thermodynamics](@ref Thermodynamics-section) section.
```